packages/query/src/stacks/token-metadata/fungible-tokens/fungible-token-metadata.hooks.ts

```diff
@@ -27,7 +27,7 @@ export function useStacksFungibleTokensBalance(
           client,
           network: network.chain.stacks.url,
         }),
-        select: (resp: FtAssetResponse) => {
+        select: (resp: unknown) => {
```

Comment:
Does this need to be `unknown`?

Reply:
That is bc you removed `client.getFtMetadata(address, signal) as unknown as FtAssetResponse` on L33 in the queryFn.

Reply:
in catch block of query fn we return null if there is api 404 err, so this won't help here

Reply:
If there are errors it would be good to try fix them if we can to avoid losing the type. Or else provide `FtAssetResponse | unknown`

Reply:
I was getting false errors in vscode... it worked ok once restarted ts server...
fixed the types to `FtAssetResponse | null`

Reply:
thanks guys 👍